sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -471,10 +477,12 @@ public partial class CopyModelOperation : Azure.Operation<Azure.AI.FormRecognize
     }
     public partial class CreateComposedModelOperation : Azure.AI.FormRecognizer.Training.CreateCustomFormModelOperation
     {
-        public CreateComposedModelOperation(string operationId, Azure.AI.FormRecognizer.Training.FormTrainingClient client) : base (default(string), default(Azure.AI.FormRecognizer.Training.FormTrainingClient)) { }
+        protected CreateComposedModelOperation() { }
+        public CreateComposedModelOperation(string operationId, Azure.AI.FormRecognizer.Training.FormTrainingClient client) { }
```

Comment:
@maririos I'm trying to understand why the explicit base constructor call was removed, but it's still not clear to me (same thing happened in `TrainingOperation`). Any idea?

Reply:
yeah.. weird. I guess the generation thinks you could use the () constructor... but... idk :|
Not blocking the PR on this, but it will be good to follow up.